hetzner/README-cloud.md

```diff
@@ -161,11 +202,12 @@ As well as the inventory entries for the server.
 ```bash
 $ ansible-playbook ansible/playbook/passstore_controller_inventory_remove.yml -e vm_name=my-name -e pass_provider=hetzner
 ```
-# Next steps
+
+## Next steps
 
 Once the server is created it must be secured before installing other software. For that check [this README file](../ansible/playbook/README.md).
 
-## Steps to create a k8s / okd cluster
+# Steps to create a k8s / okd cluster
```

Comment:
Do we still have to mention okd ?

Reply:
Removed the okd reference from the README